README.md

```diff
@@ -20,28 +20,36 @@ see [this](http://stackoverflow.com/questions/13660777/c-reading-the-data-part-o
 #include "wave/file.h"
 
 int main() {
-  // read file's content
-  wave::File read_file;
-  read_file.Open("/home/gvne/test.wav");
-  std::error_code err;
-  auto content = read_file.Read(err);
-  if (err) {
-    std::cout << "Something went wrong" << std::endl;
-    return 0;
-  }
-
-  // write that same content to another file
-  wave::File write_file;
-  write_file.Open("/home/gvne/test_write.wav");
-  write_file.set_sample_rate(read_file.sample_rate());
-  write_file.set_bits_per_sample(read_file.bits_per_sample());
-  write_file.set_channel_number(read_file.channel_number());
-
-  write_file.Write(content, err);
-  if (err) {
-    std::cout << "Something went wrong" << std::endl;
-    return 0;
-  }
+  // read file's content
+  wave::File read_file;
+  wave::Error err = read_file.Open("/home/gvne/test.wav", wave::kIn);
+  if (err) {
+    std::cout << "Something went wrong in in open" << std::endl;
+    return 1;
+  }
+  std::vector<float> content;
+  err = read_file.Read(&content);
+  if (err) {
+    std::cout << "Something went wrong in read" << std::endl;
+    return 2;
+  }
+
+  // write that same content to another file
+  wave::File write_file;
+  write_file.Open("/home/gvne/test_write.wav", wave::kOut);

```

Comment:
```suggestion
  err = write_file.Open("/home/gvne/test_write.wav", wave::kOut);
```

Reply:
Good catch!  I fixed it in my application code as well, which is where I copied this new example from.